tests/peft_integration/test_peft_integration.py

```diff
@@ -845,7 +845,7 @@ def test_peft_pipeline_no_warning(self):
         # Create pipeline with PEFT model while capturing log output
         # Check that the warning message is not present in the logs
         pipeline_logger = logging.get_logger("transformers.pipelines.base")
-        with self.assertNoLogs(pipeline_logger, logging.ERROR) as cl:
```

Comment:
maybe originally the author intends to use

> with CaptureLogger(logger) as cl

and do the check at the end just like below.

Reply:
Yes, that was the original intent, but when I tried the original test locally, it passed even without the fix, hence I suggested to change to `self.assertNoLogs`.